behave/features/steps/fala_end_to_end_tests.py

```diff
@@ -42,3 +50,34 @@ def step_impl_result_page(context, location):
     assert current_path, f"""{CLA_FALA_URL}/?postcode={location}+&name=&search="""
     assert title_xpath, f"{FALA_HEADER}"
     assert result_container_xpath, result_number_paragraph is not None
+
+
+@step(
+    'I am taken to the page corresponding to the "{location}" "{organisation}" search result'
+)
+def step_impl_result_page_with_multi_params(context, location, organisation):
+    current_path = context.helperfunc.get_current_path()
+    title_xpath = context.helperfunc.find_by_xpath("//html/body/div/main/div/div/h1")
+    result_container_xpath = context.helperfunc.find_by_xpath(
+        '//div[@class="search-results-container"]'
+    )
+    result_number_paragraph = context.helperfunc.find_by_xpath(
+        '//p[@class="govuk-body"]'
+    )
+
+    assert (
+        current_path
+    ), f"""{CLA_FALA_URL}/?postcode={location}+&name={organisation}&search="""
+    assert title_xpath, f"{FALA_HEADER}"
```

Comment:
What is this assert here? 
Is it assert title_xpath and f"{FALA_HEADER}" are both truthy?

Reply:
my assumption was that this asserts they are both the same/equal. Is this not the correct syntax for this? 🤔

Reply:
💡 Maybe this one can be rewritten as `title_xpath` is equal to `{FALA_HEADER} + location + organisation` and not only comparing it to the first part of the `url` ?

Reply:
I think it's `assert condition, [message to show when assert fails]`